.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2.1
+
+references:
+  circleci-docker-primary: &circleci-docker-primary trussworks/circleci-docker-primary:385c300cc218b910b6fd2da8041fe848001cecf4
+  postgres: &postgres circleci/postgres:12.3-ram
+
+jobs:
+  validate:
+    docker:
+      - image: *circleci-docker-primary
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - pre-commit-dot-cache-v1-{{ checksum ".pre-commit-config.yaml" }}
+      - run:
+          name: Run pre-commit tests
+          command: pre-commit run --all-files
+      - save_cache:
+          key: pre-commit-dot-cache-v1-{{ checksum ".pre-commit-config.yaml" }}
+          paths:
+            - ~/.cache/pre-commit
+  test:
+    docker:
+      - image: *circleci-docker-primary
+        environment:
+        - PGHOST: 127.0.0.1
+        - PGDATABASE: bork-local
+        - PGPORT: 5432
+        - PGUSER: postgres
+        - PGPASS: postgres
+        - APP_ENV: test
+        - API_PROTOCOL: http
+        - API_HOST: localhost
+        - API_PORT: 8080
+        - PGSSLMODE: disable
+      - image: *postgres
+        environment:
+        - POSTGRES_DB: bork-local
+        - POSTGRES_PASSWORD: postgres
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - go-mod-sources-v1-{{ checksum "go.sum" }}
+      - run: wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/6.5.3/flyway-commandline-6.5.3-linux-x64.tar.gz | tar xvz && sudo ln -s `pwd`/flyway-6.5.3/flyway /usr/local/bin 
```

Comment:
Is this a common utility we're using? It might be worth putting this in a docker container for usage.

Reply:
[There is a docker image that we can probably leverage here](https://github.com/flyway/flyway-docker), although I haven't looked into how to do that in Circle yet. Do you think that is a better approach than installing it into this image?